tests/mocks/services.py

```diff
@@ -1490,6 +1493,12 @@ async def GetFragment(self, stream: Stream[GetFragmentRequest, GetFragmentRespon
         self.fragment_id = request.id
         await stream.send_message(GetFragmentResponse(fragment=self.fragment))
 
+    async def GetFragmentHistory(self, stream: Stream[GetFragmentHistoryRequest, GetFragmentHistoryResponse]) -> None:
+        request = await stream.recv_message()
+        assert request is not None
+        self.id = request.id
```

Comment:
Can we also set `self.page_token` and `self.page_id` here so we can confirm that those get passed correctly?